### DISCUSSION TITLE
What exactly is $1=255 supposed to do?

### DISCUSSION THREAD

**Discussion by phil-barrett** [terjeio/grblHAL#151] [Q&A] [answered]
I have always been under impression that $1=255 causes enable to be on. The term I recall seeing used is "locked". Is this incorrect? With $1=255, enable is de-asserted and you can move the stepper motor axle easily.

**Comment by terjeio** [comment, marked as answer]
Build date? There is a bug in the master branch, should be ok in the test branch.

Note that if the board has separate enable pins $37 can be used to set which axes not to disable.

**Reply by phil-barrett** [reply]
Ah, yes.  thanks.
